automated-testing/synthetic-monitoring-tests/readme.md

```diff
@@ -0,0 +1,82 @@
+# Synthetic Monitoring Tests
+
+Synthetic Monitoring Tests are a set of functional tests that run alongside the core product to verify its health and resilience, at runtime.
+
+## Why Synthetic Monitoring tests [The Why]
+
+Traditionally, software providers rely on testing through CI/CD stages in the well known [testing pyramid](https://martinfowler.com/bliki/TestPyramid.html) (unit, integration, e2e) to validate that the product is healthy and without regressions. Such tests will run on the build agent or in the test/stage environment before being deployed to production and released to live user traffic. During the services' lifetime in the production environment, they are safeguarded by monitoring and alerting tools that rely on Real User Metrics/Monitoring ([RUM](https://en.wikipedia.org/wiki/Real_user_monitoring)).
+
+However, as more organizations today provide highly-available (99.9+ SLA) products, they find that the nature of long-lived distributed applications, which typically rely on several hardware and software components, is to fail. Frequent releases (sometimes multiple times per day) of various components of the system can create further instability. This rapid rate of change to the production environment tends to make testing during CI/CD stages not hermetic and actually not representative of the end user experience and how the production system actually behaves.
+
+For such systems, the ambition of service engineering teams is to reduce to a minimum the time it takes to fix errors, or the [MTTR - Mean Time To Repair](https://en.wikipedia.org/wiki/Mean_time_to_repair). It is a continuous effort, performed on the live/production system.
+
+Synthetic Monitoring tests are a subset of tests that run in production, sometimes named Test-in-Production or Shift-Right tests. Shift right compliments and add on top of the Shift-Left paradigms that are so popular, providing modern engineering teams a broader set of tools to assure high SLAs over time.
+
+## Synthetic Monitoring tests Design Blocks [The What]
+
+Testing using synthetic data and real testing accounts injects user behaviors to the system and validates their effect, usually by passively relying on existing monitoring and alerting capabilities.
```

Comment:
Could you add a definition along the lines of "A synthetic monitoring test is a test that ..."